moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/moveit_cpp.h

```diff
@@ -177,9 +177,8 @@ class MoveItCpp
   planning_scene_monitor::PlanningSceneMonitorPtr planning_scene_monitor_;
 
   // Planning
-  std::map<std::string, planning_pipeline::PlanningPipelinePtr> planning_pipelines_;
-  std::map<std::string, std::set<std::string>> groups_pipelines_map_;
-  std::map<std::string, std::set<std::string>> groups_algorithms_map_;
```

Comment:
Is this change related in any way?

Reply:
Thanks, `MoveItCpp::execute()` is the answer I was looking for.

Wouldn't it be also better to remove `getLastPlanSolution()`? I don't see the use case.

Reply:
I think being able to introspect or process the solution makes sense, but maybe it should really be non-const.